pages/background.md

```diff
@@ -1,25 +1,30 @@
+
+The historiography of pirates outside that of historical fantasy is quite thin. There is often a blurred line when discussing the acts of piracy and privateering based on who is doing the research, and who is being affected the most. The goal of our group is to understand the history in which has been written about piracy and their effect on the Triangular trade; not only in a social and cultural aspect, but more importantly how pirates effected the trade economically.
 
 ---
 
 ## Digital Background
 
-You should also make sure to do research on any relevant digital work, whether scholarly articles or digital projects. Make sure to check Digital Humanities Quarterly, or [DHQ](http://www.digitalhumanities.org/dhq/), [Debates in the Digital Humanities](http://dhdebates.gc.cuny.edu/), [JStor](https://jstor.org), blogs and more to find relevant work.
-
-You don't actually have to have a sepearte 'Digital Background' section. If it feels more natural to you, you may combine them into a single discussion. This is especially a good idea if you feel that with your topic you cannot talk about historiography without also talking about digital scholarship, and vice versa.
+At the end of April 1726, the Elizabeth, a British naval unit, was overthrown by twenty seven year old William Fly, a lowly boatswain, and the rest of the lower class crew. In this violent upheaval history marked the decline of pirate activity throughout the world. As the story of William Fly is clouded with the insatiable desire for more and more. This was Fly’s downfall, and a poetic description of piracy in the 1700’s. Fly’s story eventually ends in death, but along the way Fly had no guilt in the piracy he performed, and he did not faulter even with a noose around his neck.  There are many stories about piracy in the 1700’s however, many people would say that they are just stories and much of the narrative is elaborated to place emphasis on the fantasy of piracy.  Because of this sort of bias the digital history of pirates contains little to no actual historical evidence. In hopes of proving that pirates inflicted punishing blows to the Atlantic Slave trade we are forced to use one of very few digital databases that is available for public use. 
```

Comment:
Okay, but what about other projects trying to use data visualization to, for example, visualize information about trade in the transatlantic zone? The very point of this area was to thing of digital projects in the same way that you do historiography.

There are all kinds of articles that relate to this, both in terms of geographic/chronological theme, and also in methodology. By only considering other digital projects specifically on piracy, you don't discuss all kinds of relate project include...
(1) The very database from which you got the information... kind of critical here
(2) The fact that another student from this class also created a project on this very subject, which I shared with your group

Bit of a shortfall here